<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>442</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="verticalWidget" native="true">
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="UMLCreateCircleButton" name="circleButton">
           <property name="minimumSize">
            <size>
             <width>4</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="UML_resurses.qrc">
             <normaloff>:/images/circle_icon.png</normaloff>:/images/circle_icon.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="UMLCreateTriangleButton" name="triangleButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="UML_resurses.qrc">
             <normaloff>:/images/triangle_icon.png</normaloff>:/images/triangle_icon.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="UMLCreateRectangleButton" name="rectangleButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="UML_resurses.qrc">
             <normaloff>:/images/rectangle_icon.png</normaloff>:/images/rectangle_icon.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="midLineWidth">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="UMLCreateConnectiveLineButton" name="lineButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="UML_resurses.qrc">
             <normaloff>:/images/line_icon.png</normaloff>:/images/line_icon.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="UMLMoveHandButton" name="handButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="UML_resurses.qrc">
             <normaloff>:/images/hand_icon.png</normaloff>:/images/hand_icon.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QGraphicsView" name="graphicsView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="midLineWidth">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UMLCreateCircleButton</class>
   <extends>QPushButton</extends>
   <header>umlcreatecirclebutton.h</header>
  </customwidget>
  <customwidget>
   <class>UMLCreateRectangleButton</class>
   <extends>QPushButton</extends>
   <header>umlcreaterectanglebutton.h</header>
  </customwidget>
  <customwidget>
   <class>UMLCreateTriangleButton</class>
   <extends>QPushButton</extends>
   <header>umlcreatetrianglebutton.h</header>
  </customwidget>
  <customwidget>
   <class>UMLCreateConnectiveLineButton</class>
   <extends>QPushButton</extends>
   <header>umlcreateconnectivelinebutton.h</header>
  </customwidget>
  <customwidget>
   <class>UMLMoveHandButton</class>
   <extends>QPushButton</extends>
   <header>umlmovehandbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="UML_resurses.qrc"/>
  <include location="UML_resurses.qrc"/>
 </resources>
 <connections/>
</ui>
